example/node-middleware.cjs

```diff
@@ -1,8 +1,8 @@
-'use strict';
```

Comment:
Please revert these unrelated changes

Reply:
Done. (The checked-in file is not formatted with prettier)